app/models/user.rb

```diff
@@ -50,6 +50,13 @@ def groups
     []
   end
 
+  def as_json(*)
```

Comment:
@danhorst yes, hyrax should be changed to use JBuilder, but AFAICT that code in Hyrax is a collection of `SolrDocument`s not of `User` objects.